torch/csrc/jit/mobile/nnc/aot_compiler.cpp

```diff
@@ -0,0 +1,120 @@
+// (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+
+#include "aot_compiler.h"
+
+#include <ATen/Functions.h>
+#include <ATen/NativeFunctions.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/jit_opt_limit.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/tensorexpr/ir.h>
+#include <torch/csrc/jit/tensorexpr/ir_simplifier.h>
+#include <torch/csrc/jit/tensorexpr/kernel.h>
+#include <torch/csrc/jit/tensorexpr/llvm_codegen.h>
+#include <torch/csrc/jit/tensorexpr/loopnest.h>
+
+#include <torch/csrc/jit/passes/peephole.h>
+#include <torch/csrc/jit/passes/remove_mutation.h>
+#include <torch/csrc/jit/passes/shape_analysis.h>
+#include <torch/csrc/jit/passes/symbolic_shape_analysis.h>
+
+using namespace torch::jit;
+using namespace torch::jit::tensorexpr;
+
+namespace torch {
+namespace jit {
+namespace mobile {
+namespace nnc {
+
+std::vector<int64_t> getConstSizes(const BufPtr b) {
+  std::vector<int64_t> r;
+  for (auto dim : b->dims()) {
+    IntImmPtr int_imm_dim = to<IntImm>(dim);
+    // TODO: assert it's actually immediate
+    int64_t s = int_imm_dim->value();
+    r.push_back(s);
+  }
+  return r;
+}
+
+void get_compiled_function(
```

Comment:
Nit: even though we're not very consistent with this, I think our guidelines are to use lowerCamelCase for function names:
> * We follow the Facebook C++ Style Guidelines. Examples: some_namespace, SomeClass, SomeEnum, someFunction, someMethod, someMemberVariable, somePrivateMemberVariable_, SOME_MACRO
> * Exception: C++ methods and functions bound into python use python naming scheme (snake_case, ...) also in their C++ names. Exposing a previously unexposed method might need renaming the C++ method.

Reply:
Thanks for the review! 
Addressed the comments.